crates/l2/sequencer/l1_committer.rs

```diff
@@ -541,19 +564,51 @@ impl GenServer for L1Committer {
 
     type Error = CommitterError;
 
+    // Right now we only have the `Commit` message, so we ignore the `message` parameter
     async fn handle_cast(
         &mut self,
         _message: Self::CastMsg,
         handle: &GenServerHandle<Self>,
     ) -> CastResponse {
-        // Right now we only have the Commit message, so we ignore the message
         if let SequencerStatus::Sequencing = self.sequencer_state.status().await {
-            let _ = self
-                .commit_next_batch_to_l1()
+            let current_last_committed_batch = self
+                .eth_client
+                .get_last_committed_batch(self.on_chain_proposer_address)
                 .await
-                .inspect_err(|err| error!("L1 Committer Error: {err}"));
+                .unwrap_or(self.last_committed_batch);
+            let Some(current_time_ms) = utils::system_now_ms() else {
```

Comment:
I'd avoid using `_ms` suffix outside of struct attributes
```suggestion
            let Some(current_time) = utils::system_now_ms() else {
```

Reply:
Done in [eb3653e](https://github.com/lambdaclass/ethrex/pull/4099/commits/eb3653e05a22d73c43225c58ead4c88dfe4d3882)